From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sat, 1 Sep 2018 17:14:01 -0400
Subject: Use system-wide PYTHONPATH for elpy-rpc-pythonpath

---
 elpy.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elpy.el b/elpy.el
index d22e2c1..367873b 100644
--- a/elpy.el
+++ b/elpy.el
@@ -238,7 +238,7 @@ and not an interactive shell like ipython."
           (member val '("python" "python2" "python3" "pythonw")))
   :group 'elpy)
 
-(defcustom elpy-rpc-pythonpath (file-name-directory (locate-library "elpy"))
+(defcustom elpy-rpc-pythonpath nil
   "A directory to add to the PYTHONPATH for the RPC process.
 
 This should be a directory where the elpy module can be found. If
