From: galaunay <gaby.launay@tutanota.com>
Date: Tue, 17 Apr 2018 17:03:02 +0100
Subject: Prevent 'expand-file-name' from erroring when project-root is nil

---
 elpy.el | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/elpy.el b/elpy.el
index b0e4256..26d1312 100644
--- a/elpy.el
+++ b/elpy.el
@@ -2118,8 +2118,9 @@ root directory."
   "Common routine for formatting python code."
   (let ((line (line-number-at-pos))
         (col (current-column))
-        (directory (or (expand-file-name (elpy-project-root))
-                              default-directory)))
+        (directory (if (elpy-project-root)
+                       (expand-file-name (elpy-project-root))
+                     default-directory)))
     (if (use-region-p)
         (let ((new-block (elpy-rpc method
                                    (list (elpy-rpc--region-contents)
