From: Romain Leroux <lerouxrgd@users.noreply.github.com>
Date: Tue, 24 Mar 2020 03:22:14 +0100
Subject: Fix a fatal typo in elpy-format-code

---
 elpy.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/elpy.el b/elpy.el
index f2631f5..dc8aa1f 100644
--- a/elpy.el
+++ b/elpy.el
@@ -2279,10 +2279,10 @@ prefix argument is given, prompt for a symbol from the user."
     (when (interactive-p) (message "Autoformatting code with yapf."))
     (elpy-yapf-fix-code))
    ((elpy-config--package-available-p "autopep8")
-    ((when (interactive-p) message "Autoformatting code with autopep8."))
+    (when (interactive-p) (message "Autoformatting code with autopep8."))
     (elpy-autopep8-fix-code))
    ((elpy-config--package-available-p "black")
-    ((when (interactive-p) message "Autoformatting code with black."))
+    (when (interactive-p) (message "Autoformatting code with black."))
     (elpy-black-fix-code))
    (t
     (message "Install yapf/autopep8 to format code."))))
