From: Jorgen Schaefer <contact@jorgenschaefer.de>
Date: Fri, 20 Apr 2018 09:24:32 +0000
Subject: Make coding system explicit for files with umlauts

---
 test/elpy-rpc-test.el | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/elpy-rpc-test.el b/test/elpy-rpc-test.el
index cb49885..8b5bdca 100644
--- a/test/elpy-rpc-test.el
+++ b/test/elpy-rpc-test.el
@@ -1,3 +1,5 @@
+;;; -*-coding: utf-8-*-
+
 (ert-deftest elpy-rpc-should-call-async-with-success-callback ()
   (elpy-testcase ()
     (mletf* ((elpy-rpc--call
