From: Jorgen Schaefer <contact@jorgenschaefer.de>
Date: Fri, 20 Apr 2018 09:37:36 +0000
Subject: Typo fix

---
 elpy-django.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elpy-django.el b/elpy-django.el
index cfe77e3..92732e6 100644
--- a/elpy-django.el
+++ b/elpy-django.el
@@ -252,7 +252,7 @@ This requires Django 1.6 or the django-discover-runner package."
 (put 'elpy-test-django-runner 'elpy-test-runner-p t)
 
 (define-minor-mode elpy-django
-  "Minor mode to for Django commands."
+  "Minor mode for Django commands."
   :group 'elpy-django)
 
 (provide 'elpy-django)
